### ISSUE TITLE
Cannot import on Windows

### ISSUE THREAD

**Issue by cHemingway** [afg984/python-cxxfilt#6]
Attempting to import the library, with Python 3.8.1, Windows 10, gets the following error.

``` 
>>> import cxxfilt
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\christopher.h\AppData\Local\Programs\Python\Python38-32\lib\site-packages\cxxfilt\__init__.py", line 39, in <module>
    libc = ctypes.CDLL(find_any_library('c'))
  File "C:\Users\christopher.h\AppData\Local\Programs\Python\Python38-32\lib\site-packages\cxxfilt\__init__.py", line 36, in find_any_library
    raise Error('Cannot find any of libraries: {}'.format(choices))
cxxfilt.Error: Cannot find any of libraries: ('c',)
```

Checking the [ctypes documentation](https://docs.python.org/3/library/ctypes.html#finding-shared-libraries), [line 33](https://github.com/afq984/python-cxxfilt/blob/c4aa26224b0bc8e1556c20cf1fefe6fd0eef5e96/cxxfilt/__init__.py#L33) is not valid on Windows, as:
> On Windows, find_library() searches along the system search path, and returns the full pathname, but since there is no predefined naming scheme a call like find_library("c") will fail and return None.

Instead, it appears [`find_mscvrt()`](https://docs.python.org/3/library/ctypes.html#ctypes.util.find_msvcrt) should be used on windows, especially as libc is only being used for `free()`

**Comment by cHemingway** [comment]
Looking more closely at this, it seems [UnDecorateSymbolName](https://docs.microsoft.com/en-us/windows/win32/api/dbghelp/nf-dbghelp-undecoratesymbolname) should be used instead of `__cxa_demangle()` on Windows.

**Comment by afq984** [comment]
Yes, windows [is not supported](https://github.com/afq984/python-cxxfilt/blob/master/README.rst#supported-environments) now. I'm not familiar with using Python in Windows so there are a few questions to answer:
1. Does `UnDecorateSymbolName` use the same mangling rules as libstdc++/libc++? If not, what would a user expect this library to do? (update: Should it demangle using libstdc++'s rules or using MSVC's rules?)
2. Need to set up a CI for windows.

**Comment by afq984** [comment]
The `import` time error will be solved by #9.
The error will be delayed to `demangle` `demangleb` calls.
Open another issue if actual windows support is needed.
